File: .codex-error-screenshot.py
from pathlib import Path

from playwright.sync_api import sync_playwright


output = Path(r"D:\Proyecto Electrodental NUEVO\electrodental-error-view.png")
output.parent.mkdir(parents=True, exist_ok=True)

with sync_playwright() as p:
    browser = p.chromium.launch(headless=True)
    page = browser.new_page(viewport={"width": 1440, "height": 980}, device_scale_factor=1)
    page.goto("http://localhost:8080/ruta-para-probar-error", wait_until="networkidle")
    page.screenshot(path=str(output), full_page=True)
    browser.close()

print(output)

File: .codex-error-screenshot.js
const { chromium } = require("C:/Users/santi/.cache/codex-runtimes/codex-primary-runtime/dependencies/node/node_modules/.pnpm/playwright@1.61.1/node_modules/playwright");

(async () => {
  const output = "C:/tmp/electrodental-error-view.png";
  const browser = await chromium.launch({ channel: "msedge", headless: true });
  const page = await browser.newPage({ viewport: { width: 1440, height: 980 }, deviceScaleFactor: 1 });
  await page.goto("http://localhost:8080/ruta-para-probar-error", { waitUntil: "networkidle" });
  await page.screenshot({ path: output, fullPage: true });
  await browser.close();
  console.log(output);
})();
